versions.tf

```
@@ -4,19 +4,11 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 5.0.0"
-    }
-    kubernetes = {
-      source  = "hashicorp/kubernetes"
-      version = ">= 2.7.1"
+      version = ">= 5.33.0"
```

```suggestion
      version = ">= 5.34.0"
```
Mentioned at https://github.com/cloudposse/terraform-aws-eks-cluster/issues/203#issuecomment-1901668621, due to https://github.com/hashicorp/terraform-provider-aws/pull/35391

---

Good catch!